packages/core/docs/general/context.md

```diff
@@ -1,6 +1,12 @@
 # Context API
```

Comment:
imho this is 100% content for either integrations category od a new one like "Internals"